user_storage_postgresql.py

```diff
@@ -1,50 +1,64 @@
+import pg8000
 from user import User
 from datetime import datetime, timezone
-from postgresql_singleton import PostgreSQLSingleton
+from bot.config_reader import env_config
 from typing import List
 
 
+def get_connection():
+    return pg8000.connect(
+        user=env_config.postgresql_username,
+        password=env_config.postgresql_password.get_secret_value(),
+        host=env_config.postgresql_hostname,
+        port=env_config.postgresql_port,
+        database=env_config.postgresql_database,
+    )
+
+
 class UserStoragePostgreSQL:
 
     def get_all_users(self) -> List[User]:
-        result = PostgreSQLSingleton.getConnection().run("SELECT * FROM users")
-
-        users = []
-        for row in result:
-            user = User(
-                telegram_id=row[0],
-                first_name=row[1],
-                last_name=row[2],
-                username=row[3],
-                created_at=row[4],
-                updated_at=row[5],
-            )
-            users.append(user)
-        print(users)
-        return users
+        with get_connection() as connection:
```

Comment:
После правки в get_connection (см. комментарий выше), тут достаточно будет сделать:
```suggestion
        connection = get_connection()
```